shared/src/business/useCases/users/generateChangeOfAddress.test.js

```diff
@@ -147,4 +204,55 @@ describe('generateChangeOfAddress', () => {
       isAutoGenerated: true,
     });
   });
+
+  it('should notify honeybadger and continue processing the second user-case if the first user-case is invalid', async () => {
```

Comment:
@ericsorenson another Honeybadger notification - will this work in the court environment?